script.js

```diff
@@ -0,0 +1,111 @@
+let calcDisplay = document.querySelector("#calc-display");
+let butID = document.querySelectorAll("button")
+let prevNum = "";
+let nextNum = 0;

```

Comment:
be consistent in the data types you will save. your data as, since the dom values will be strings stay with 2 strings. 

if you will convert to number before saving then stick with 2 number vals as the initial